src/plugins/intel_cpu/src/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -1934,6 +1934,7 @@ void MKLDNNEltwiseNode::prepareParams() {
     // together with the corresponding appendPostOps method to pass the scales and shifts pointers at runtime.
     // Until then we have to read them from the quantization_t directly, store them somewhere
     // and nullify them to get read of the address dependency in the key structure
+    fqDataPtrs.clear();
```

Comment:
@maxnick , I think we need reset fqDataPtrs first here to remove fqDataPtrs of previously stored, as it's always passed from beginning element[(args_ptrs.post_op_data = fqDataPtrs.data())](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/mkldnn_eltwise_node.cpp#L1999) for current execution. Do you agree?

Reply:
Yes, of course!!! Nice catch!